include/seqan3/alignment/multiple/detail.hpp

```diff
@@ -66,6 +67,30 @@ auto convert_alph_3_to_2(seqan3::aa10li chr)
     return seqan::ReducedAminoAcid<seqan::Li10>{seqan3::to_char(chr)};
 }
 
+/*!
+ * \brief Predicate that represents whether `candidate_t` is a type that is not allowed as MSA configuration.
+ * \tparam candidate_t The type to test.
+ *
+ * \details
+ * The predicate's value is set to: NOT (band OR gap OR scoring).
+ */
+template <typename candidate_t>
+struct is_invalid_msa_config :
+    std::negation<std::disjunction<std::is_same<candidate_t, seqan3::align_cfg::band_fixed_size>,
```

Comment:
Why do we need `std::negation<std::disjunction..` ? Can be explain to me what it does, I've never seen this before :)

Maybe a `seqan3::type_list` with `seqan3::list_traits::find_if` also suffices?

Reply:
I tried to make it clear with the comment above: 
We want to return `std::true_type` (which means invalid) if `candidate_t` is any type except the 3 allowed ones. The disjunction is an OR operation, so if one of the 3 checks equals `true_type` then the disjunction is `true_type`, and in exactly this case we want to return `false_type` (so we need `negation`).

**Example 1**: `candidate_t` = `gap`
result_type = !(false_type || true_type || false_type) = !true_type = false_type

**Example 2**: `candidate_t` = `parallel`
result_type = !(false_type || false_type || false_type) = !false_type = true_type

We cannot use `find_if` here, because it would return `bool` instead of `std::bool_constant` (and we need to represent a type, not a value, because the predicate in the `find_if` statement below requires that).

Reply:
Thanks for the explanation :)